common/persistence/persistence-tests/visibilityPersistenceTest.go

```diff
@@ -43,7 +43,7 @@ type (
 		// override suite.Suite.Assertions with require.Assertions; this means that s.NotNil(nil) will stop the test,
 		// not merely log an error
 		*require.Assertions
-		VisibilityMgr             p.VisibilityManager
```

Comment:
make sure run `make lint` and `make fmt` to reformat our code. Otherwise buildkite will fail.